chero/InputParser.cs

```diff
@@ -31,12 +31,15 @@ public static List<MoveAction> parse()
                     // preperate string for further processing
                     string output = Regex.Replace(line, @"\d+\.\s*", string.Empty);
                     List<string> substrings = new List<string>(output.Split(' '));
+                    substrings.Remove(substrings.Last());
                     List<MoveAction> ret = new List<MoveAction>();
                     int counter = 0;
                     bool isWhite;
 
                     foreach (string substring in substrings)
                     {
+                        string substr = substring;
```

Comment:
Gute Idee die nicht benötigten zeichen zu entfernen. Aber warum brauchen wir hier die neue variable "substr"? Wenn ich das richtig verstehe würde das genauso funktionieren, wenn wir die Zeile weglassen und das gleiche mit "substring" machen, die wir in der Zeile drüber im foreach-block definiert haben.